Eastern-Research-Group/mywaterway
# Feature/499 all waterbodies popups

## Related Issues:
* [HMW-499](https://jira.epa.gov/browse/HMW-499)

## Main Changes:
* Fixed issue of popups not working for the all waterbodies layer after being saved.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Open the Add/Save data widget
3. Ensure the "All Waterbodies" and "Waterbodies" layers are toggled on
4. Save the layer
5. Open the newly saved web map in AGO
6. Verify popups work for both the "All Waterbodies" and "Waterbodies" layers
